packages/slice-machine/pages/cts/[ct].tsx

```diff
@@ -49,11 +49,22 @@ const CustomTypeBuilderWithRouter = () => {
     })
   );
 
+  const { cleanupCustomTypeStore } = useSliceMachineActions();
+
   useEffect(() => {
     if (!selectedCustomType || !hasLocal(selectedCustomType))
       void router.replace("/");
   }, [selectedCustomType, router]);
 
+  useEffect(() => {
```

Comment:
I don't think you need the router events, just returning a function that executes cleanupCustomTypeStore is enough.

Reply:
You're right! Will update :)